src/modules/Rotate/info.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Rotate",
+    "description": "Rotates image by specified degrees",
+    "inputs": {
+      "rotate": {
+        "type": "integer",
+        "desc": "Angular value for rotation",
```

Comment:
Can you specify "in degrees" or "in radians" etc? Thank you!!! Then, i think this is ready for merge -- what do you think?

Reply:
yess sure ! I have updated 😄